src/ftest.jl

```diff
@@ -1,42 +1,48 @@
-type FTestResult
-    ntests::Int
-    SSR::Array{Float64, 1}
-    nparams::Array{Int, 1}
-    dof_resid::Array{Int, 1}
-    R2::Array{Float64, 1}
-    fstat::Array{Float64, 1}
-    pval::Array{PValue, 1}
+type FTestResult{N}
+    ssr::NTuple{N, Float64}
+    nparams::NTuple{N, Int}
+    dof_resid::NTuple{N, Int}
+    r2::NTuple{N, Float64}
+    fstat::Tuple{Vararg{Float64}}
+    pval::Tuple{Vararg{PValue}}
 end
 
-#function FTestResult{N}(SSR1::Array{Float64, 1}, fstati
-"""A helper function to determine if mod1 is nested in mod2"""
-function issubmodel(mod1::LinPredModel, mod2::LinPredModel)
-    mod1.rr.y != mod2.rr.y && return false # Response variables must be equal
-
-    # Now, test that all predictor variables are equal
-    pred1 = mod1.pp.X
-    npreds1 = size(pred1, 2)
-    pred2 = mod2.pp.X
-    npreds2 = size(pred1, 2)
-    # If model 1 has more predictors, it can't possibly be a submodel
-    npreds1 > npreds2 && return false 
-    
-    @inbounds for i in 1:npreds1
-        var_in_mod2 = false
-        for j in 1:npreds2
-            if view(pred1, :, i) == view(pred2, :, j)
-                var_in_mod2 = true
-                break
-            end
-        end
-        if !var_in_mod2
-            # We have found a predictor variable in model 1 that is not in model 2
-            return false 
-        end
-    end
-
-    return true
-end
+ #function FTestResult{N}(SSR1::Array{Float64, 1}, fstati
+                 """A helper function to determine if mod1 is nested in mod2"""
+                 function issubmodel(mod1::LinPredModel, mod2::LinPredModel)
+                 mod1.rr.y != mod2.rr.y && return false # Response variables must be equal
+
+ # Now, test that all predictor variables are equal
+                 pred1 = mod1.pp.X
+                 npreds1 = size(pred1, 2)
+                 pred2 = mod2.pp.X
+                 npreds2 = size(pred1, 2)
+ # If model 1 has more predictors, it can't possibly be a submodel
+                 npreds1 > npreds2 && return false 
+
+                 @inbounds for i in 1:npreds1
+                 var_in_mod2 = false
+                 for j in 1:npreds2
+                 if view(pred1, :, i) == view(pred2, :, j)
+                 var_in_mod2 = true
+                 break
+                 end
+                 end
+ if !var_in_mod2
+ # We have found a predictor variable in model 1 that is not in model 2
+ return false 
+ end
+ end
+
+ return true
+ end
+
+_diffn{N, T}(t::NTuple{N, T})::NTuple{N, T} =  ntuple(i->t[i]-t[i+1], N-1)
+
+_diff{N, T}(t::NTuple{N, T})::NTuple{N, T} =  ntuple(i->t[i+1]-t[i], N-1)
+
+import Base: ./
+./{N, T1, T2}(t1::NTuple{N, T1}, t2::NTuple{N, T2}) = ntuple(i->t1[i]/t2[i], N)
```

Comment:
Unfortunately you can't do that as that will affect other packages. You can just use `ntuple` directly, or define a custom function.